src/MountedTraversal.js

```diff
@@ -47,6 +47,9 @@ export function instEqual(a, b, lenComp) {
 
 export function instHasClassName(inst, className) {
   const node = findDOMNode(inst);
+  if (!node) { // inst renders null
```

Comment:
If the instance renders "nothing", it could return an empty array, or `null`, or `undefined`… is there perhaps a react method we could use, like `React.Children.count` or something, that would cover us here?